git-sync/main.go

```diff
@@ -40,6 +40,9 @@ var flWait = flag.Int("wait", envInt("GIT_SYNC_WAIT", 0), "number of seconds to
 var flOneTime = flag.Bool("one-time", envBool("GIT_SYNC_ONE_TIME", false), "exit after the initial checkout")
 var flDepth = flag.Int("depth", envInt("GIT_SYNC_DEPTH", 0), "shallow clone with a history truncated to the specified number of commits")
 
+var flAllowFail = flag.Bool("allow-failures", envBool("GIT_SYNC_ALLOW_FAILURES", false), "allow temporal git failures (the first pull must succeed)")
```

Comment:
Do we need two flags?  why not just --max-sync-failures=0 means none allowed, and any other value means they are allowed.

I'm a fan of very specific flag names, `max-sync-failures` ?